requirements/base.txt

```diff
@@ -12,7 +12,6 @@ django-cors-headers==3.2.1  # via -r requirements/base.in (line 7)
 django-haystack==2.8.0    # via -r requirements/base.in (line 5)
 django-rest-swagger==2.1.2  # via -r requirements/base.in (line 4)
 django-waffle==0.20.0     # via edx-django-utils, edx-drf-extensions
-django==1.11.28           # via -r requirements/base.in (line 1), django-cors-headers, django-haystack, djangorestframework, edx-django-release-util, edx-django-utils, edx-drf-extensions, rest-condition
```

Comment:
Why was the django dependency removed from this file?  Should have only been removed from `test.txt`.